cdap-kubernetes/src/main/java/io/cdap/cdap/master/environment/k8s/KubeMasterEnvironment.java

```diff
@@ -182,7 +152,7 @@ public String getName() {
 
   @Override
   public Optional<MasterEnvironmentTask> getTask() {
-    return Optional.ofNullable(podKillerTask);
+    return Optional.empty();
```

Comment:
Just remove this method. The parent is already having a default implementation that returns an `Optional.empty()`

Reply:
+1